WalletWasabi.Daemon/Config/Config.cs

```diff
@@ -138,7 +138,10 @@ [ nameof(ExchangeRateProvider)] = (
 				GetStringValue("ExchangeRateProvider", PersistentConfig.ExchangeRateProvider, cliArgs)),
 			[ nameof(FeeRateEstimationProvider) ] = (
 				"The mining fee rate estimation provider. Available providers are (default) MempoolSpace and BlockstreamInfo",
-				GetStringValue("FeeRateEstimationProvider", PersistentConfig.FeeRateEstimationProvider, cliArgs))
+				GetStringValue("FeeRateEstimationProvider", PersistentConfig.FeeRateEstimationProvider, cliArgs)),
+			[ nameof(DropUnconfirmedTransactionsAfterDays) ] = (
```

Comment:
feel free to ignore:
something shorter might be better, also on the console
![image](https://github.com/user-attachments/assets/bc6d6a68-f351-4e7e-804e-e1fc9ca93729)

maybe `MaxDaysUnconfirmedTransaction`